charts/node-local-dns/templates/configmap.yaml

```diff
@@ -95,7 +95,11 @@ data:
       {{- end }}
       {{- if $v.plugins.health }}
       {{- if not $ciliumConfig }}
+      {{- if contains ":" $localDnsIp }}
+      health [{{ $localDnsIp }}]:{{ $v.plugins.health.port }}
```

Comment:
I think we can approach this by removing `$localDnsIp` from the `health` plugin address and use just the port to support both versions, right?